bin/hdfcoinc/pycbc_coinc_hdfinjfind

```diff
@@ -206,14 +206,14 @@ for trigger_file, injection_file in zip(args.trigger_files,
                    sim_table.get_column(args.redshift_column))
 
     if multi_ifo_style:
-        for key in f.keys():
+        for key in f['segments'].keys():
             if key == 'foreground':
                 continue
             if key not in fo:
                  fo.create_group(key)
-            fo[key].attrs['pivot'] = f[key].attrs['pivot']
-            fo[key].attrs['fixed'] = f[key].attrs['fixed']
-            fo[key].attrs['foreground_time_exc'] = f[key].attrs['foreground_time_exc']
+            fo[key].attrs['pivot'] = f['segments/%s' % key].attrs['pivot']
```

Comment:
does this make a group called key which just has the attributes hung to it and nothing else? I think this is okay but wanted to be sure

Reply:
If the key does not exists, it is created.

Reply:
yes to me it seems only the attributes are there specific to the detector combination